src/main/archetype/ui.content/src/main/content/jcr_root/conf/__confFolderName__/settings/wcm/templates/content-page/structure/.content.xml

```diff
@@ -9,6 +9,15 @@
         <root
             jcr:primaryType="nt:unstructured"
             sling:resourceType="wcm/foundation/components/responsivegrid">
+            <languagenavigation
+                jcr:primaryType="nt:unstructured"
+                sling:resourceType="${appsFolderName}/components/content/languagenavigation"/>
+            <search
+                jcr:primaryType="nt:unstructured"
+                sling:resourceType="${appsFolderName}/components/content/search"/>
+            <navigation
+                jcr:primaryType="nt:unstructured"
+                sling:resourceType="${appsFolderName}/components/content/navigation"/>
```

Comment:
Also setting navigationRoot pointing to contentRoot would makes sense, or WDYT?
--